src/SipClient.ts

```diff
@@ -0,0 +1,256 @@
+import {
+  SIPTrunkInfo,
+  CreateSIPTrunkRequest,
+  ListSIPTrunkRequest,
+  ListSIPTrunkResponse,
+  DeleteSIPTrunkRequest,
+  SIPDispatchRule,
+  SIPDispatchRuleInfo,
+  CreateSIPDispatchRuleRequest,
+  ListSIPDispatchRuleRequest,
+  ListSIPDispatchRuleResponse,
+  DeleteSIPDispatchRuleRequest,
+  SIPParticipantInfo,
+  CreateSIPParticipantRequest,
+} from '@livekit/protocol';
+import ServiceBase from './ServiceBase.js';
+import { livekitPackage, Rpc, TwirpRpc } from './TwirpRPC.js';
+import {
+  SIPDispatchRuleDirect,
+  SIPDispatchRuleIndividual,
+} from '@livekit/protocol/src/gen/livekit_sip_pb';
```

Comment:
I think we should export this directly from the protocol package's index file, as this pattern might cause errors for some picky bundlers.

Reply:
adding `SIPDispatchRuleDirect, SIPDispatchRuleIndividual` to the import from `@livekit/protocol` should work.

Reply:
Done!